### DISCUSSION TITLE
Capture mime-types and supported extensions as part of the `storage` configuration

### DISCUSSION THREAD

**Discussion by dcousens** [keystonejs/keystone#8913] [Feature requests]
As discussed in https://github.com/keystonejs/keystone/pull/6715

In `@keystone-6/core <= 5.7.0`, unfortunately we are quite opinionated about what mime-types we support as part of the `file` and `image` fields.

We should update the configuration to support a user-prescribed list of mime types, in a way that the users can apply their project's security considerations and restrictions.
We should have some warnings around any common pitfalls, but that should be documentation.
 
 ### Existing Signal 📶 
- https://github.com/keystonejs/keystone/pull/6715
- https://github.com/keystonejs/keystone/issues/8538
- https://github.com/keystonejs/keystone/discussions/8203

**Comment by erreib** [comment]
Was curious if there had been any changes or movement on this? Or alternatively if anyone had implemented a custom field that allowed for svg uploads?
